ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/PeerPendingTransactionTracker.java

```diff
@@ -67,6 +68,10 @@ public synchronized void addToPeerSendQueue(final EthPeer peer, final Hash hash)
     }
   }
 
+  public boolean isPeerSupported(final EthPeer peer) {
```

Comment:
The protocol should be an argument, for future feature support.  I don't think this will be the first time we will change behavior based on ETH subprotocol support.

Reply:
:+1: